docs/maps/search.asciidoc

```diff
@@ -0,0 +1,37 @@
+[role="xpack"]
+[[maps-search]]
+== Searching Your Data
+
+The **Maps** application embeds the query bar with autocomplete for real-time ad hoc search.
+All layers with {es} sources are updated when you submit a search request.
+
+image::maps/images/global_search_bar.png[]
+
+
+[float]
+=== Layer with {es} source
+
+You can create a layer with {es} source from the following:
+
+- <<vector-layer>> with *Documents* source.
+
+- <<vector-layer>> with *Grid aggregation* source.
+
+- <<heatmap-layer>> with *Grid aggregation* source.
+
+- <<vector-layer>> with <<terms-join>>.
+
+
+[float]
+=== Searching across multiple indexes
+
+Your map may contain multiple {es} indexes.
+This can occur when your map contains two or more layers with {es} sources from different indexes.
+This can also occur with a single layer with an {es} source and a <<terms-join>>.
+
```

Comment:
It would be nice to show an example search here, before talking about empty layers.